metadata-ingestion/setup.py

```diff
@@ -150,7 +150,7 @@ def get_long_description():
     # PyAthena is pinned with exact version because we use private method in PyAthena
     "athena": sql_common | {"PyAthena[SQLAlchemy]==2.4.1"},
     "azure-ad": set(),
-    "bigquery": sql_common | bigquery_common | {"sqlalchemy-bigquery>=1.4.1"},
+    "bigquery": sql_common | bigquery_common | usage_common | {"sqlalchemy-bigquery>=1.4.1"},
```

Comment:
I think it is a bit misleading we name it as usage_common and we use it with non-usage sources

Reply:
Any suggestions?

Reply:
maybe add the sqlparser directly? 
my concern is that from now on all the additional usage_common dependencies will automatically be a dependency of the bigquery source as well. 
Which will be fine if we merge usage with source.

Reply:
changed